### PR TITLE
Mention context/route

### PR DESCRIPTION
**Pull request by kyrylo**
`context/route` is displayed in Error Overview. When an account uses APM, the
route will be linkified. The link will lead to the APM dashboard.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -267,7 +267,9 @@ curl -X POST -H "Content-Type: application/json" -d JSON "https://airbrake.io/ap
       "id": "12345",
       "name": "root",
       "email": "root@root.com"
-    }
+    },
+
+    "route": "/pricing"
```
Comment (zefer) [inline source/index.html.md]: Does this normally also include the HTTP method?

Comment (kyrylo) [reply source/index.html.md]: Nope, it works without a method.

Comment (zefer) [reply source/index.html.md]: I know it works, but _normally_ we include the method don't we? A good example would be close to the normal usage.

Comment (kyrylo) [reply source/index.html.md]: Good point, yeah, I think this is paired normally.

PTAL https://github.com/airbrake/slate/pull/59

**Pull request merged**